--- slackConfig14.1ROOT.sh	2014-08-26 07:14:07.639946483 -0700
+++ slackConfigROOT.sh	2014-08-26 07:14:14.318946026 -0700
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# wget -N https://raw2.github.com/ryanpcmcquen/config-o-matic/master/slackConfig14.1ROOT.sh; sh slackConfig14.1ROOT.sh; rm slackConfig14.1ROOT.sh
+# wget -N https://raw2.github.com/ryanpcmcquen/config-o-matic/master/slackConfigROOT.sh; sh slackConfigROOT.sh; rm slackConfigROOT.sh
 
 ## BASHGITVIM="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/bashGitVimROOT.sh"
 
@@ -8,12 +8,12 @@
 SBOPKGDL="http://sbopkg.googlecode.com/files/sbopkg-0.37.0-noarch-1_cng.tgz"
 SPPLUSDL="http://sourceforge.net/projects/slackpkgplus/files/slackpkg%2B-1.3.2-noarch-1mt.txz"
 
-SPPLUSMATECONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/14.1/mate/slackpkgplus.conf"
-SPPLUSMLIBCONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/14.1/multilib/slackpkgplus.conf"
-SPPLUSCONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/14.1/slackpkgplus.conf"
+SPPLUSCONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/current/slackpkgplus.conf"
+SPPLUSMATECONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/current/mate/slackpkgplus.conf"
+SPPLUSMLIBCONF64="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/64/current/multilib/slackpkgplus.conf"
 
-SPPLUSMATECONF32="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/32/14.1/mate/slackpkgplus.conf"
-SPPLUSCONF32="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/32/14.1/slackpkgplus.conf"
+SPPLUSCONF32="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/32/current/slackpkgplus.conf"
+SPPLUSMATECONF32="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/32/current/mate/slackpkgplus.conf"
 
 INSCRPT="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/slackware/initscript"
 
@@ -27,7 +27,7 @@
 
 TOUCHPCONF="https://raw2.github.com/ryanpcmcquen/linuxTweaks/master/51-synaptics.conf"
 
-GETEXTRA="https://raw.githubusercontent.com/ryanpcmcquen/linuxTweaks/master/slackware/getExtraSlackBuildsSTABLE.sh"
+GETEXTRA="https://raw.githubusercontent.com/ryanpcmcquen/linuxTweaks/master/slackware/getExtraSlackBuildsCURRENT.sh"
 
 CALPLAS="Caledonia-1.9.tar.gz"
 CALWALL="Caledonia_Official_Wallpaper_Collection-1.5.tar.gz"
@@ -37,9 +37,7 @@
 
 if [ ! $UID = 0 ]; then
   cat << EOF
-
 This script must be run as root.
-
 EOF
   exit 1
 fi
@@ -137,15 +135,6 @@
 fi
 
 
-wget -N $BASHRC -P ~/
-wget -N $BASHPR -P ~/
-wget -N $VIMRC -P ~/
-
-## set tmux scrollback value
-tmux set-option -g history-limit 9999
-## set to xterm otherwise vi will break
-tmux set-option -g default-terminal xterm-color
-
 ## configure lilo
 sed -i 's/^#compact/lba32\
 compact/g' /etc/lilo.conf
@@ -162,20 +151,32 @@
 sed -i 's/^export LANG=en_US/#export LANG=en_US/g' /etc/profile.d/lang.sh
 sed -i 's/^#export LANG=en_US.UTF-8/export LANG=en_US.UTF-8/g' /etc/profile.d/lang.sh
 
+## adjust slackpkg blacklist
+sed -i 's/^aaa_elflibs/#aaa_elflibs/g' /etc/slackpkg/blacklist
+
 sed -i 's/#\[0-9]+_SBo/\
 \[0-9]+_SBo\
 sbopkg/g' /etc/slackpkg/blacklist
 
-### undo current
+### undo 14.1 mirrors
 sed -i \
-'s_^http://ftp.osuosl.org/.2/slackware/slackware-current/_# http://ftp.osuosl.org/.2/slackware/slackware-current/_g' /etc/slackpkg/mirrors
+'s_^http://ftp.osuosl.org/.2/slackware/slackware-14.1/_# http://ftp.osuosl.org/.2/slackware/slackware-14.1/_g' /etc/slackpkg/mirrors
 sed -i \
-'s_^http://ftp.osuosl.org/.2/slackware/slackware64-current/_# http://ftp.osuosl.org/.2/slackware/slackware64-current/_g' /etc/slackpkg/mirrors
+'s_^http://ftp.osuosl.org/.2/slackware/slackware64-14.1/_# http://ftp.osuosl.org/.2/slackware/slackware64-14.1/_g' /etc/slackpkg/mirrors
 ### osuosl mirrors are stable and fast (they are used for the changelogs), choose mirrorbrain if you are far from oregon
 sed -i \
-'s_^# http://ftp.osuosl.org/.2/slackware/slackware-14.1/_http://ftp.osuosl.org/.2/slackware/slackware-14.1/_g' /etc/slackpkg/mirrors
+'s_^# http://ftp.osuosl.org/.2/slackware/slackware-current/_http://ftp.osuosl.org/.2/slackware/slackware-current/_g' /etc/slackpkg/mirrors
 sed -i \
-'s_^# http://ftp.osuosl.org/.2/slackware/slackware64-14.1/_http://ftp.osuosl.org/.2/slackware/slackware64-14.1/_g' /etc/slackpkg/mirrors
+'s_^# http://ftp.osuosl.org/.2/slackware/slackware64-current/_http://ftp.osuosl.org/.2/slackware/slackware64-current/_g' /etc/slackpkg/mirrors
+
+wget -N $BASHRC -P ~/
+wget -N $BASHPR -P ~/
+wget -N $VIMRC -P ~/
+
+## set tmux scrollback value
+tmux set-option -g history-limit 9999
+## set to xterm otherwise vi will break
+tmux set-option -g default-terminal xterm-color
 
 ## git config
 git config --global user.name "$GITNAME"
@@ -190,7 +191,7 @@
 wget -N https://raw.githubusercontent.com/ryanpcmcquen/linuxTweaks/master/slackware/sbo/hooks/pre-commit \
   -P /usr/share/git-core/templates/hooks/
   ## this is just in case permissions are incorrect,
-  ## these files should already be 755  ;^)
+  ## these files should already be 755 ;^)
   chmod 755 /usr/share/git-core/templates/hooks/*
 
 wget -N $TOUCHPCONF -P /etc/X11/xorg.conf.d/
@@ -201,6 +202,7 @@
   wget -N $SPPLUSDL -P ~/
 fi
 installpkg ~/*.t?z
+mv /etc/slackpkg/slackpkgplus.conf /etc/slackpkg/slackpkgplus.conf.old
 rm ~/*.t?z
 
 ## set slackpkg to non-interactive mode to run without prompting
@@ -492,3 +494,5 @@
 echo
 
 
+
+
